library/psa_crypto_ffdh.c

```diff
@@ -10,6 +10,12 @@
 
 #if defined(MBEDTLS_PSA_CRYPTO_C)
 
+/* This header is only needed because it defines
+ * MBEDTLS_DHM_RFC7919_FFDHExxxx_[P|G]_BIN symbols that are used in
```

Comment:
I checked, and apparently we need XXXX (uppercase) in order to make `check_names.py` happy.